src/Db/Adapter/AdapterInterface.php

```diff
@@ -51,6 +51,7 @@ interface AdapterInterface
     public const PHINX_TYPE_JSON = 'json';
     public const PHINX_TYPE_JSONB = 'jsonb';
     public const PHINX_TYPE_UUID = 'uuid';
+    public const PHINX_TYPE_NATIVEUUID = 'nativeuuid';
```

Comment:
I'm hoping to remove all of these constants in the future and use the abstract types defined in CakePHP. We'll have to make sure that the same abstract type names are used so that we don't break compatibility.